database/models/User.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  DataType,
+  Column,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+  BelongsToMany,
+} from "sequelize-typescript"
+
+import { Site } from "./Site"
+import { SiteMember } from "./SiteMember"
+
+@Table({ tableName: "users" })
+export class User extends Model {
+  @Column({
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+    type: DataType.BIGINT,
+  })
+  id!: number
+
+  @Column({
+    allowNull: true,
+    unique: true,
+    type: DataType.TEXT,
+    validate: {
+      isEmail: true,
```

Comment:
i think this is already validated in `newroutes/users` ([here](https://github.com/isomerpages/isomercms-backend/pull/364/files#diff-f5a20552f742c7a8229e2a99ba078eaee60527809c656c1678f0cbba5458bfebR19-R20)) so it might be helpful to just drop a comment here. (they're the exact same validation underneath as sequelize also uses `validators.js`, see [here](https://sequelize.org/master/manual/validations-and-constraints.html))

Reply:
Will fix